Podfile.lock

```diff
@@ -59,6 +59,7 @@ PODS:
   - HockeySDK (5.1.4):
     - HockeySDK/DefaultLib (= 5.1.4)
   - HockeySDK/DefaultLib (5.1.4)
+  - JTAppleCalendar (8.0.2)
```

Comment:
`Podfile.lock` still needs to be updated. I got local changes again, this time on `Giphy`.

Reply:
What's the practice you generally use for this? I used `pod update JTAppleCalendar` to avoid changes to other pods since the previous changes seemed like they were unwanted in this PR. If I delete the lockfile and re-run `pod install` I get quite a few updates. If I run `pod install` I see only this one.

Reply:
@ScoutHarris Ignore this. I ended up getting the new Zendesk checksum. Not sure why it didn't work the first few times.